rules/docker-tam.mk

```diff
@@ -0,0 +1,29 @@
+# docker image for TAM
```

Comment:
Why is it called docker TAM?
AFAIK this container implements a localhost TAM collector, so maybe the name should be more specific?

Reply:
The docker is intended to host multiple TAM applications in near future. Currently, it only hosts  thresholdmgr. Since the docker is intended to host TAM applications, the docker is named docker-tam.